### DISCUSSION TITLE
Any way to map tilde?

### DISCUSSION THREAD

**Discussion by jcnorkus** [kriomant/ch57x-keyboard-tool#66] [Q&A]
I'm trying to map tilde to one of the keys, I've tried all the arbitrary scan codes and I wasn't able to figure it out. Has anyone tried or figured it out or even know how to do this? I've even tried both below.

This validates but the key just prints the letter "a"
`- ["f13", "f14", ~]`

and this fails to validate entirely
`- ["f13", "f14", "~"]`

**Comment by kriomant** [comment]
`shift+grave`.
